lib/iris/tests/integration/test_grib2.py

```diff
@@ -36,16 +36,13 @@
 import iris.tests.stock as stock
 from iris.util import is_regular
 
-# Grib support is optional.
+# gribapi is an optional dependency
 if tests.GRIB_AVAILABLE:
     try:
-        # Try to load the independent 'iris_grib' package.
-        from iris_grib import load_pairs_from_fields
-        from iris_grib.message import GribMessage
-    except ImportError:
-        # Try to load old inbuilt module instead.
         from iris.fileformats.grib import load_pairs_from_fields
         from iris.fileformats.grib.message import GribMessage
+    except ImportError:
+        pass
```

Comment:
@lbdreyer If `GRIB_AVAILABLE == True` then why wrap this all in a `try ... except` block?